operator/internal/manifests/internal/config/loki-config.yaml

```diff
@@ -182,6 +182,32 @@ ingester_client:
 # Values for not exposed fields are taken from the grafana/loki production
 # configuration manifests.
 # (See https://github.com/grafana/loki/blob/main/production/ksonnet/loki/config.libsonnet)
+{{- if and .Stack.PatternIngester .Stack.PatternIngester.Enabled }}
+pattern_ingester:
+  enabled: true
+  lifecycler:
+    ring:
+      heartbeat_timeout: 1m
+      replication_factor: {{ .Stack.Replication.Factor }}
+      kvstore:
+        store: memberlist
+    num_tokens: 512
+    heartbeat_period: 5s
+    join_after: 30s
```

Comment:
I don't think this section is necessary from the testing I did. Also `replication_factor` cannot be other than `1` for pattern-ingester https://github.com/JoaoBraveCoding/loki/blob/main/pkg/pattern/ingester.go#L168-L170

Reply:
Noted. I'll do one more round of testing to verify this.

Reply:
You were right, no need for the `lifecycler:` section.. updating PR now